generate/generate.go

```diff
@@ -143,12 +149,15 @@ func New() Generator {
 				},
 			},
 			Devices: []rspec.Device{},
-		},
+		}
+	} else {
+		return generator, fmt.Errorf("no defaults configured for %s", os)
```

Comment:
I think you need to generate the appropriate configuration for `windows` and `solaris` .

Reply:
> I think you need to generate the appropriate configuration for `windows` and `solaris`.

I'm creating a very slim Solaris default with 984db97c.  I'm happy to add a Window default too if you'll tell me what it should be, but I don't know enough about Windows to be able to figure that out on my own.  Since we don't currently support generating defaults for Windows configs, I don't see a problem with leaving that unimplemented for this PR and just adjusting the function signatures to allow it to be added in the future.